app/webpacker/packs/school_search.js

```diff
@@ -0,0 +1,32 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+import fetch from "node-fetch";
```

Comment:
The package for this is https://www.npmjs.com/package/whatwg-fetch, node-fetch is something backend JS!

You won't need to import anything, too - just add `import 'whatwg-fetch'` to application.js and then use `fetch` in this file as it's a browser API.

Reply:
Done